### PR TITLE
Next geojson popups

### LINKED ISSUE #320: Fix display of GeoJson popups
Right now they are displayed like this:

Screenshot from 2020-11-26 16-21-10

### PR DESCRIPTION
**Pull request by meromisi**
## Proposed Changes

  - fix geojson popup content
  - fix geojson popup position

image

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform

closes #320

### CODE REVIEW

File: app/configurations/config.hbnext.js [hunk 1]
```diff
@@ -180,7 +180,13 @@ export default configMerger(walttiConfig, {
         showLayerSelector: true, // DT-3470
         showStopMarkerPopupOnMobile: false, // DT-3470
         showScaleBar: true, // DT-3470, DT-3397
-
+        genericMarker: {
+            popup: {
+                offset: [0,0],
+                maxWidth: 250,
```
Comment (hbruch) [inline app/configurations/config.hbnext.js:186]: Is maxWidth==minWidth reasonable? Or will we see truncated text then?

Comment (meromisi) [reply app/configurations/config.hbnext.js:186]: It is actually not necessary, with or without them they look the same. I just thought its better if they have the same size but if you want I can remove it. The important property is the `offset` so the popup is not shifted from the icon.

**Comment by leonardehrenfried** [comment]
@hbruch If you're happy with the PR, please merge.

**Comment by leonardehrenfried** [comment]
I took the liberty to merge this. Hope that's ok.

**Pull request merged**